src/styles.css

```diff
@@ -54,26 +48,33 @@ body {
 /* NAV BAR */
 nav {
   display: flex;
-  justify-content: flex-end;
-  align-items: flex-start;
-  width: 100%;
+  justify-content: space-between;
+}
+
+nav > h1 {
+  text-align: center;
+  margin-left: 25px;
+  cursor: pointer;
 }
 
 .nav-child {
   display: flex;
-  justify-content: space-around;
   align-items: center;
-  width: 13%;
-  margin: 2%;
+  margin-right: 25px;
   cursor: pointer;
 }
 
+#searchArea {
+  padding-right: 100px;
+}
+
 .nav-icon {
   height: 1.75rem;
   background-color: white;
   opacity: 0.4;
   border-radius: 50%;
-  margin: 0 auto;
+  margin: 0 2px 0 2px;
+  padding: 2px;
```

Comment:
Could the padding please be increased for nav-icons? I tried 5px padding and it looks pleasing without looking bloated. Right now I personally felt that the container for a nav icon felt crowded.